src/fetch/payouts.py

```diff
@@ -99,19 +93,15 @@ def __init__(  # pylint: disable=too-many-arguments, too-many-positional-argumen
     @classmethod
     def from_series(cls, frame: Series) -> RewardAndPenaltyDatum:
         """Constructor from row in Dataframe"""
-        slippage = (
-            int(frame["eth_slippage_wei"])
-            if not math.isnan(frame["eth_slippage_wei"])
-            else 0
-        )
+        slippage = int(frame["slippage_eth"]) + int(frame["network_fee_eth"])
         solver = frame["solver"]
         reward_target = frame["reward_target"]
-        if pandas.isna(reward_target):
+        if pd.isna(reward_target):
```

Comment:
I would slightly edit the msg to write `Using solver submission address instead.`